services/movie.py

```diff
@@ -6,14 +7,16 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
typehint:
bool | None = None

Reply:
to be honest i've not catch the idea.. This method returns queryset in fact? I've redone but it looks strange a bit..

Reply:
@phaishuk if argument accept default as None then type must be Optional
In this case Optional[str]

Reply:
Check also in other places please)

Reply:
@ArsenPidhoretskyi aha, okay redoing